nym-node-status-api/nym-node-status-agent/Cargo.toml

```diff
@@ -1,7 +1,6 @@
 # Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
 # SPDX-License-Identifier: Apache-2.0
 
-
 [package]
 name = "nym-node-status-agent"
 version = "1.0.0-rc.1"
```

Comment:
Just a side question. What is the versioning and release process for these?

Reply:
discussed, closing